addons/mail/models/mail_channel_rtc_session.py

```diff
@@ -48,13 +48,25 @@ def unlink(self):
                 # Note: invitation depends on field `rtc_inviting_session_id` so the cancel must be
                 # done before the delete to be able to know who was invited.
                 channel._rtc_cancel_invitations()
-        self.env['bus.bus'].sendmany([((self._cr.dbname, 'mail.channel', channel.id), {
+        notifications = [((self._cr.dbname, 'mail.channel', channel.id), {
             'type': 'rtc_sessions_update',
```

Comment:
It's slightly out-of-scope of this PR, but I feel `rtc_sessions_update` should eventually be removed.

In the deletion flow, ideally there should only 1 notification to broadcast that the RTC sessions have ended (no 2 currently). And maybe creation should make dedicate notification for `rtc_sessions_created`.

Ideally the payload shouldn't be contextual data on the thread, but more an `insert` or `delete` on the RTC Session record (and data should properly link/unlink to channel).

But again, slightly out-of-scope of PR, just that it's more obvious now that there's no more `_disconnect` method.